evaluation/generation/generate.py

```diff
@@ -0,0 +1,57 @@
+import argparse
+import datetime
+
+import torch
+from transformers import AutoTokenizer, AutoModelForCausalLM
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--checkpoint", type=str, help="Checkpoint path", required=True)
+    parser.add_argument("--parallelize", action="store_true")
+    parser.add_argument("--global-step", type=str, default=None)
+    parser.add_argument("--generate-max-length", type=int, default=50, help="max generation length")
+    parser.add_argument("--greedy", action="store_true")
+    parser.add_argument("--top-k", type=int, default=0)
+
+    return parser.parse_args()
+
+def generate_from_text(model, text, tokenizer, max_length=200, greedy=False, top_k=0):
+    input_ids = tokenizer.encode(text, return_tensors='pt').to("cuda:0")
+    max_length = input_ids.size(-1) + max_length
+    
+    greedy_output = model.generate(
+        input_ids.to('cuda:0'),
+        max_length=max_length,
+        do_sample=not greedy,
+        top_k=None if greedy else top_k,
+    )
+    return tokenizer.decode(greedy_output[0], skip_special_tokens=True)
+
+def main():
+    args = get_args()
+    print(f"Loading model")
+
+    tokenizer = AutoTokenizer.from_pretrained(args.checkpoint, padding_side="left")
```

Comment:
I think based on your discussion on slack, we can now leave the padding_side as the default which is "right"?
Since the below is the same
```
from transformers import AutoModelForCausalLM, AutoTokenizer

tokenizer = AutoTokenizer.from_pretrained("bigscience/bloom-1b3")

tokenizer.padding_side = "right"

model = AutoModelForCausalLM.from_pretrained(
    "bigscience/bloom-1b3",
)

inputs = ["Hello", "Joe Biden is the president of"]
inputs_right = tokenizer(inputs, return_tensors="pt", padding=True)

tokenizer.padding_side = "left"

inputs_left = tokenizer(inputs, return_tensors="pt", padding=True)
outputs_right = model.generate(**inputs_right)
outputs_left = model.generate(**inputs_left)

print(tokenizer.decode(outputs_left[0], skip_special_tokens=True))
print(tokenizer.decode(outputs_right[0], skip_special_tokens=True))

```